zingolib/src/wallet/data.rs

```diff
@@ -456,12 +460,12 @@ pub struct ReceivedOrchardNoteAndMetadata {
     pub note: orchard::note::Note,
 
     // The postion of this note's commitment
```

Comment:
Let's rewrite this comment.   "commitment to the funding value for this note" perhaps?